### DISCUSSION TITLE
[#991931] Can't look around when used a portal in crosshair mode

### DISCUSSION THREAD

**Discussion by TORD31** [Mojang/minecraft-editor#66] [Bugs]
When you used a nether or end portal in crosshair mode you can't look around anymore and you move the cursor instead.

Repro steps
1. Go in crosshair mode
2. build a nether or endportal
3. Enter the portal
4. See that you can't look around but move your cursor

Observed result vs. what you actually expected to happen
After you entered the portal you can't look around and you move your curser instead until you go to tool mode and than back to crosshair mode. I expect that you can look normaly around after entered the portal without changing the mode like it is in normal minecraft.

In this video I demonstrate the issue: https://1drv.ms/v/s!Asr1UnK1Sz_LgQN9FljCVTb96h-V?e=xOEzuj

**Comment by minecraftchloe** [comment]
Great find!!! Updated title with internal tracking ID.
